### PR TITLE
docs: add some examples to some comps

### PR DESCRIPTION
**Pull request by amyspark-ng**
yea

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -368,7 +368,18 @@ export interface KAPLAYCtx<
      *
      * @param a The angle to rotate by. Defaults to 0.
```
Comment (dragoncoder047) [inline src/types.ts:371]: might be worthwhile to note that angles for this are in degrees, some other game libs use radians

Comment (lajbel) [reply src/types.ts:371]: true

Comment (amyspark-ng) [reply src/types.ts:371]: doesn't it already say (in degrees)

Comment (dragoncoder047) [reply src/types.ts:371]: oh duh, it was the line that got cut off by the GitHub diff

**Comment by amyspark-ng** [comment]
if it's good then push pls

**Comment by niceEli** [comment]
conflicts fixed

**Review by niceEli — APPROVED** [review]

**Pull request merged**